### DISCUSSION TITLE
Kata 2

### DISCUSSION THREAD

**Discussion by CrisGutierrez05** [LaunchX-InnovaccionVirtual/CursoIntroPython#270] [Q&A] [answered]
Hola, disculpen a mi me aparece esto despues de crear el entorno virtual y no se que hacer, me podrían ayudar por favor?
![Captura de pantalla (610)](https://user-images.githubusercontent.com/88925198/154031870-76849541-932f-48a4-b0ea-9a6a7bc46afc.png)

**Comment by ArielCanedo** [comment]
Aquí puedes probar
env\Scripts\activate

Env es el nombre del entorno virtual, así que si cambia el nombre el
comando será (nombre)\Scripts\activate

**Reply by CrisGutierrez05** [reply]
Hola, así se activó el entorno, pero en donde estoy confundio es en la parte de creación donde debo poner **python3 -m venv env** y me aparece No module named venv.

**Reply by carlos-gmx3** [reply]
El entorno no está activado
Cuando lo está te lo indica agregando (env) previo a la ruta en la terminal
El comando venv configura un directorio con todo lo necesario para ejecutar las aplicaciones de python sin intepretes externos al mismo directorio, primero checa si python --version funciona. si no es posible que debas agregar la ruta a las variables de entorno o descargar Python desde la tienda de Microsoft
Te quedaría algo así despues de activar el entorno virtual
`(env) C:\Users\admin\Documents...`
prueba

**Reply by CrisGutierrez05** [reply]
Muchas gracias, ya note lo que estaba mal, era la diagonal ponía esta / en vez de esta \.

**Comment by ArielCanedo** [comment, marked as answer]
revisa que tengas instalado Python con
Python --version

Luego crea el entorno virtual con python -m venv env
Lo activas con env\Scripts\activate

**Reply by CrisGutierrez05** [reply]
Muchas gracias.
